rfcs/20210510-conda-forge-distribution/README.md

```diff
@@ -0,0 +1,52 @@
+# Conda-forge distribution
+
+## Introduction
+
+Conda is a python package manager that widely used by ML FWKs users
+
+## Conda channels
+- default. This is a channel managed by Anaconda core team.
+  oneDNN is available as part of Intel oneAPI [distribution](https://anaconda.org/intel/repo).
+- forge. Conda forge is a community led Conda channel that consists of build
+  infrastructure and distributions for the Conda package manager. Recently oneDNN became
+  available in [Conda-forge](https://github.com/conda-forge/onednn-feedstock) channel.
+
+The main differences between Conda and Conda-forge:
+- Conda-forge has broader architecture support;
+- Conda-forge has more packages available;
+- To provide binary compatibility between packages in Conda-forge all package
+  dependencies should be packaged by Conda-forge, meaning there is no way to
+  distribute oneDNN built by Intel C++ Compiler without having this compiler
+  in Conda-forge;
+- And [more](https://conda-forge.org/docs/user/introduction.html#why-conda-forge).
+
+## oneDNN usage cases. Summary
+
+|          | Runtime package        | Build time package     | Full bundle  |
+| :------- | :--------------------- | :--------------------- | :----------- |
+| Scenario | Used as a dependency for binary distributions | Used as a dependency for source code distributions that are dependent on oneDNN. | Covers remaining cases |
+| Channel  | **conda**, pip, OS repos | **conda**, pip, OS repos | Intel oneAPI |
+| Library  | x                      | x                      | x            |
+| Headers  |                        | x                      | x            |
+| Examples |                        |                        | x            |
+| Doc      |                        |                        | x            |
```

Comment:
Should we put readme/license files?

Reply:
There is a type of license, not the license itself in the [feedstock](https://github.com/conda-forge/onednn-feedstock/blob/master/README.md#about-onednn) and in the distribution:
 
$ conda info onednn
onednn 2.2.2 tbb_h7aa542a_0
file name   : onednn-2.2.2-tbb_h7aa542a_0.tar.bz2
name        : onednn
version     : 2.2.2
build string: tbb_h7aa542a_0
build number: 0
channel     : https://conda.anaconda.org/conda-forge/linux-64
...
**license     : Apache-2.0
license_family: APACHE**

Reply:
I was thinking about all mentions about third party code/software that we have, I'm not sure it's a problem here. I think @vpirogov can comment on that.

Reply:
This should be covered by including LICENSE and THIRD-PARTY-PROGRAMS files with every package. As long as we do this we are good.

Reply:
Yes, they are there in each archive, so we are good.
I will update the RFC. Thanks for the question.